common/lifecyleMgr.go

```diff
@@ -400,15 +408,19 @@ func (lcm *lifecycleMgr) processOutputMessage() {
 	for {
 		msgToPrint := <-lcm.msgQueue
 
-		switch lcm.outputFormat {
-		case EOutputFormat.Json():
-			lcm.processJSONOutput(msgToPrint)
-		case EOutputFormat.Text():
-			lcm.processTextOutput(msgToPrint)
-		case EOutputFormat.None():
+		if shouldQuietMessage(msgToPrint, lcm.OutputVerbosityType) {
 			lcm.processNoneOutput(msgToPrint)
```

Comment:
You should discard the message here and continue rather than passing message to processNoneOutput

Reply:
No actually that would lead to our code never ending. When the code ends, it sends a message with eOutputMessageType.Error() or eOutputMessageType.EndOfJob(), and that's how we know when to call 
`lcm.closeFunc();
os.Exit(int(msgToOutput.exitCode))`

If we just discard the message, that would never be triggered, and our code would hang

Reply:
For all other eOutputMessageType (s) I could very well have discarded the message, but I have kept it for the times when it is exit or error

Reply:
`func (lcm *lifecycleMgr) processNoneOutput(msgToOutput outputMessage)`
This function will automatically discard the rest of the types

Reply:
closing this